openpype/hosts/blender/hooks/pre_set_workfile_publish_time.py

```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+
+from openpype.lib import PreLaunchHook
+from openpype.hosts.blender import utility_scripts
+
+
+class PreSetWorkfilePublishTime(PreLaunchHook):
+
```

Comment:
To avoid testing `host_name == "blender"` in the other hook, this would be both more efficient and more explicit:
```suggestion
     app_groups = [
             "blender",
         ]
```